packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js

```diff
@@ -440,8 +442,8 @@ export default class WebRTCMediaEngine {
             }
             this.logger.warn(err);
           }
-          this.logger.info(`setting receiving${capitalize[kind]} to false`);
-          this[`sending${capitalize[kind]}`] = false;
+          this.logger.info(`setting sending${capitalize[kind]} to ${constraint}`);
+          this[`sending${capitalize[kind]}`] = constraint;
```

Comment:
It will hit, but it looks like `constraint` will always be false. I could switch it back to hard coded boolean, but I like that we know where the value is coming from.